common/config/data.js

```diff
@@ -15,14 +15,21 @@ export const ANNOUNCEMENT_MESSAGE = `
   If you're interested in recieving updates about the MyEtherWallet V4 Alpha, you can subscribe via <a href="http://myetherwallet.us16.list-manage.com/subscribe?u=afced8afb6eb2968ba407a144&id=15a7c74eab">mailchimp</a> :)
 `;
 
-export const DONATION_ADDRESSES_MAP = {
+const etherScan = 'https://etherscan.io';
+const blockChainInfo = 'https://blockchain.info';
+const ethPlorer = 'https://ethplorer.io';
+
+export const ETHTxExplorer = `${etherScan}/tx/[[txHash]]`;
+export const BTCTxExplorer = `${blockChainInfo}/tx/[[txHash]]`;
+export const ETHAddressExplorer = `${etherScan}/address/[[address]]`;
+export const ETHTokenExplorer = `${ethPlorer}/address/[[address]]`;
```

Comment:
Would probably be less error-prone if you instead made these functions that return a string that flow could validate. Ex.
```js
const ETHTxExplorer = (txHash: string) => `${etherScan}/tx/${txHash}`;
```

Reply:
Yeah agreed, this is legacy leftover from V3. I'll adjust it.